stdlib/domain.mli

```diff
@@ -39,27 +39,25 @@ val self : unit -> id
 (** [self ()] is the identifier of the currently running domain *)
 
 val at_first_spawn : (unit -> unit) -> unit
```

Comment:
I wonder if we should consider naming this function `before_first_spawn` instead of `at_first_spawn`, to clarify that it is called on the parent domain, unlike `at_each_spawn` that runs on the child domain.

Reply:
`before_first_spawn` is more appropriate. Renamed.